### DISCUSSION TITLE
re-authorisation

### DISCUSSION THREAD

**Discussion by christorpher** [Budibase/budibase#9057] [Help]
i have field where i wanna get confirmation from a user as a login would have general user multiple users can confirm that specific field can i keep a field for id password on the particular form which will be prefilled to verify and put the check id password for verification
there be multiple id & password will be created for verification
is it possible in budibase

**Comment by christorpher** [comment]
Please help

**Comment by andz-bb** [comment]
Hi @christorpher, I don't think I understand what you are trying to do, could you send some screenshots or an app export to demonstrate what you are asking about? Thanks

**Comment by christorpher** [comment]
![Capture](https://user-images.githubusercontent.com/120565326/209142878-58e8dfcb-9949-4e7a-9bad-cd90effca9ca.PNG)

**Reply by christorpher** [reply]
when someone verifies they should have a field of password so they can fill in so we can know the person who exactly verified it as the app will be open from a common user name but people will have there id to confirm the verification

**Reply by christorpher** [reply]
Please let me know if it's possible

**Reply by christorpher** [reply]
Can I get a reply please
